development/tools/cmd/dnscollector/README.md

```diff
@@ -0,0 +1,59 @@
+# IP Address and DNS Record Garbage Collector
+
+## Overview
+
+This command finds and removes orphaned IP Addresses and related DNS records created by GKE integration jobs in a Google Cloud Platform (GCP) project.
+
+When an integration job installs Kyma on the GKE cluster, an IP Address and DNS records are created as well.
```

Comment:
```suggestion
When an integration job installs Kyma on a GKE cluster, an IP Address and DNS records are created as well.
```

Reply:
Does it create the IP addresses? Or does it allocate them/reserve them?

Reply:
You are right, more technically correct wording would be:
```When an integration job installs Kyma on a GKE cluster, it reserves the required IP addresses and adds necessary DNS records.```